spanner/spec/spanner_samples_spec.rb

```diff
@@ -17,27 +17,44 @@
 require "google/cloud/spanner"
 
 describe "Google Cloud Spanner API samples" do
-  before do
+  before :all do
```

Comment:
could you try `before :each` here? CI looks like it isn't skipping when it should

Reply:
maybe break out some of the other setup stuff into a `before :all` block if it doesn't need to run before each test individually